Solutions/Flare/Analytic Rules/FlareCredentialLeaks.yaml

```diff
@@ -0,0 +1,25 @@
+id: 467cbe7e-e6d4-4f4e-8f44-84dd01932c32
+name: Flare Leaked Credentials
+description: |
+    'Searches for Flare Leaked Credentials'
+severity: medium
+requiredDataConnectors:
+  - connectorId: Flare
+    dataTypes:
+      - Firework_CL
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+  - BruteForce
+relevantTechniques:
+  - T1207
+query: |
+  Firework_CL
```

Comment:
There are some errors in the KQL Validation as the validator does not understand the table name. Can we please add a json that defines the schema of the Firework_CL table at the below location. The folder has examples of other custom tables. 
[![image](https://user-images.githubusercontent.com/37783395/209928940-c510307b-a6a3-4384-8ddc-13fc63c75a90.png)](https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables)

Reply:
The schema has hundreds of possible columns and depending on the type of data, the schema will differ. do I need to add all of them here ?

Reply:
I added a file, but another error pops up saying my column names are not valid... 
```
  Failed Kqlvalidations.Tests.KqlValidationTests.Validate_DetectionQueries_HaveValidKql(fileName: "FlareCredentialLeaks.yaml", encodedFilePath: "L2hvbWUvdnN0cy93b3JrLzEvcy9Tb2x1dGlvbnMvRmxhcmUvQW"...) [113 ms]
  Error Message:
   Template Id: 9cb7c337-f170-4af6-b0e8-b6b7552d762d is not valid in Line: 2 col: 40
                    Errors: The name 'source_s' does not refer to any known column, table, variable or function., Code: 'KS142', Severity: 'Error', Location: '51..59'
```